packages/core/test/unit/data-types/binary-types.test.ts

```diff
@@ -8,6 +8,7 @@ describe('DataTypes.BLOB', () => {
     'ibmi db2': 'BLOB(1M)',
     mssql: 'VARBINARY(MAX)',
     postgres: 'BYTEA',
+    cockroachdb: 'BYTES',
```

Comment:
What's the difference between the postgres BYTEA and the cockroach BYTES?

Reply:
Sorry, it's BYTEA is an alias to BYTES, I guess I should remove the data type and use postgres implementation.